ruby/BUILD.bazel

```diff
@@ -19,6 +21,46 @@ ruby_library(
     ],
 )
 
+java_export(
+    name = "protobuf_java_release",
```

Comment:
We already have a really similar target to this in //ruby/lib/google:protobuf_lib.  Do we need this additional jar?

Reply:
That's a ruby library, but I need to use this rule to correctly package the maven jar including creating some template files.

Reply:
`protobuf_lib` is a ruby library, but under jruby it has a data dependency on `//ruby/src/main/java:protobuf_java` which is built using the resolved jruby's jar files.  There is also a jar created as an intermediary at `//ruby/lib/google/protobuf:protobuf_java.jar`